mmcv/device/ipu/model_wrapper.py

```diff
@@ -50,11 +52,11 @@ class WrappedNet(nn.Module):
     """
 
     def __init__(self,
-                 model,
-                 inputs_manager,
-                 outputs_manager,
-                 inter_outputs_in_cpu,
-                 modules_to_record=None):
+                 model: nn.Module,
+                 inputs_manager: HierarchicalDataManager,
+                 outputs_manager: HierarchicalDataManager,
+                 inter_outputs_in_cpu: dict,
+                 modules_to_record: Optional[Union[Config, List]] = None):
```

Comment:
It could be better to add `optional` in docstring(modules_to_record (`mmcv.Config or list, optional`))

Reply:
done